Sources/_OpenAPIGeneratorCore/Translator/Operations/OperationDescription.swift

```diff
@@ -144,10 +144,29 @@ extension OperationDescription {
         )
     }
 
-    /// Returns parameters from both the path item level
-    /// and the operation level.
+    /// Merged parameters from both the path item level and the operation level.
+    /// If duplicate parameters exist, only the parameters from the operation level are preserved.
+    ///
+    /// - Returns: An array of merged path item and operation level parameters without duplicates.
+    /// - Throws: When an invalid JSON reference is found.
     var allParameters: [UnresolvedParameter] {
-        pathParameters + operation.parameters
+        get throws { 
+            var mergedParameters = [UnresolvedParameter]()
```

Comment:
```suggestion
            var mergedParameters: [UnresolvedParameter] = []
```

Nit: prefer a literal.